foundation/foundation-configuration-store/src/test/java/util/ConfigStoreInitTestHelper.java

```diff
@@ -0,0 +1,138 @@
+package util;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Configuration::Store
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.configuration.store.ConfigLoader;
+import com.boozallen.aissemble.configuration.store.Property;
+import io.quarkus.runtime.Startup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.DumperOptions;
+import org.yaml.snakeyaml.Yaml;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.spi.CDI;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * ConfigStoreInit loads configuration properties as application starts up
+ */
+@Startup
+@ApplicationScoped
+public class ConfigStoreInitTestHelper {
+    private static final Logger logger = LoggerFactory.getLogger(ConfigStoreInitTestHelper.class);
+
+    @PostConstruct
+    void init() {
```

Comment:
Q: Can we extend the actual init class and re-use the logic there so we don't have a separate mirrored copy of the logic specific to the tests?

Reply:
great point 👍